recipes/kronprod/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "kronprod" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    url: https://github.com/Jbwasse2/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: 2fefedf5d3077a88a77aeb237d7b0ef3c1e91b645ecbe92db5047a5fddd1e10c 
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py<34]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - scipy
```

Comment:
Do you need `scipy` here? These are only the dependencies that are required to run the `setup.py` script.
```suggestion
```